packages/app/src/elements/org-settings.ts

```diff
@@ -169,4 +220,64 @@ export class OrgSettingsView extends Routing(StateMixin(LitElement)) {
             </div>
         `;
     }
+
+    private _renderDirectorySettings() {
+        const org = this._org!;
+
+        let scimGroupsUrl = "";
+        let scimUsersUrl = "";
+
+        const syncEnabled = org.directory.syncProvider !== "none";
+
+        if (syncEnabled && org.directory.scim) {
+            const scimSecret = bytesToBase64(org.directory.scim.secret, true);
+            scimGroupsUrl = `${org.directory.scim.url}/Groups?org=${org.id}&token=${scimSecret}`;
+            scimUsersUrl = `${org.directory.scim.url}/Users?org=${org.id}&token=${scimSecret}`;
```

Comment:
Two things:
1. I think it would be better if the urls were fully constructed on the server side. I.e. instead of `scim.url` we should have `scim.groupsUrl` and `scim.usersUrl`. The url structure is something that the client should not have to concern itself with, and harcoding this into the client will make it a lot harder to change the url structure in the future if we need to. Which brings me to my second point
2. I think for the url I this better: `${baseUrl}/${orgId}/${"users" | groups}/?token=${scimSecret}`

Reply:
Agreed!